internal/translayer/dbuser/translation.go

```diff
@@ -0,0 +1,139 @@
+package dbuser
+
+import (
+	"fmt"
+	"sort"
+
+	"go.mongodb.org/atlas-sdk/v20231115008/admin"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/pointer"
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/timeutil"
+	akov2 "github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api/v1"
+)
+
+type User struct {
+	akov2.AtlasDatabaseUserSpec
+	Password  string
+	ProjectID string
+}
+
+func toK8sDatabaseUser(dbUser *admin.CloudDatabaseUser) (*User, error) {
```

Comment:
Naming seems inconsistent. here, it is called `toK8sDatabaseUser`, below `toAtlas`. one is returning an error but not the other.

My hypothesis is that, especially in the translation layer the conversion between types needs to have a well defined contract.

Reply:
done